### PR TITLE
ADD: ATLAS Topics

### PR DESCRIPTION
**Pull request by Marrkson**
This MR adds Bachelor & Master topics in association with ATLAS

### CODE REVIEW

**Comment by leloup314** [comment]
Nice, will there be a separate PR for the introduction to ATLAS? ;)

File: topics/atlas.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+***
+
+### Topic: Characterization of ATLAS Pixel Readout Chip 
+
+(_Last modified on_: {{ "2019-10-10 00:00:00 +0000" | date: '%B %d, %Y' }})
```
Comment (leloup314) [inline topics/atlas.md]: Correct date

File: topics/atlas.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+- Type: Bachelor/Master
+- Location: SiLab
+- Contact: [Mark Standke](mailto:standke@physik.uni-bonn.de)
+- Description: ATLAS will receive an upgrade in 2026, Silab is involved in the design, verification & production of the pixel chips & modules. You can contribute by evaluation the performance of the latest ATLAS pixel chip performance, at low temperatures or after x-ray exposure. Get in touch for further info.
+
+### Topic: Systematic Analysis of Wafer-probing Data
+
+(_Last modified on_: {{ "2019-10-10 00:00:00 +0000" | date: '%B %d, %Y' }})
```
Comment (leloup314) [inline topics/atlas.md]: Correct date

**Review by leloup314 — CHANGES_REQUESTED** [review]

**Comment by Marrkson** [comment]
Done.

**Review by leloup314 — APPROVED** [review]
Cool and good

**Pull request merged**